src/backend/tests/it/utils/pocketic.rs

```diff
@@ -68,6 +216,8 @@ pub fn upgrade_with_wasm(
 
     let arg = encoded_arg.unwrap_or(encode_one(&init_arg()).unwrap());
 
+    pic.advance_time(Duration::from_secs(100_000));
```

Comment:
If we redeploy frequently without advancing the clock, `PocketIc` starts throwing errors about too many cycles being used.  I put this here rather than at the end of the builder deployment because (in theory) a canister can be redeployed many times.  In current tests we redeploy only once, as far as I can see, but this location seems safer.

Reply:
How was the value chosen? Would 1 second also be enough?
Maybe leave a comment in the code as well, so that one does not need to dig up the PR to get to this insight. 😉

Reply:
I could not find the throttling period quickly so I chose a big number (this is just over a day).  I'll put a comment in saying that this is wildly conservative and can be reduced.